central/plugins/org.jboss.tools.central/src/org/jboss/tools/central/editors/browser/VersionedBrowser.java

```diff
@@ -0,0 +1,66 @@
+/*************************************************************************************
+ * Copyright (c) 2008-2015 Red Hat, Inc. and others.
+ * All rights reserved. This program and the accompanying materials 
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ * 
+ * Contributors:
+ *     JBoss by Red Hat - Initial implementation.
+ ************************************************************************************/
+
+package org.jboss.tools.central.editors.browser;
+
+import org.eclipse.swt.browser.Browser;
+import org.eclipse.swt.widgets.Composite;
+
+/**
+ * 
+ * @author Konstantin Marmalyukov
+ *
+ */
+
+public class VersionedBrowser extends Browser {
+	private String name;
+	private String version;
+	
+	public VersionedBrowser(Composite parent, int style) {
+		super(parent, style);
+		if (System.getProperty("os.name", "generic").toLowerCase().indexOf("win") >= 0) { //$NON-NLS-1$ //$NON-NLS-2$ //$NON-NLS-3$
+			//workaround for https://bugs.eclipse.org/bugs/show_bug.cgi?id=465822
+			//this bug should be fixed into 4.6
+			setUrl(getUrl());
+		}
+		String browserScript =
+				   "var ua = navigator.userAgent,tem,M=ua.match(/(opera|chrome|safari|firefox|msie|trident(?=\\/))\\/?\\s*(\\d+)/i) || [];" +  //$NON-NLS-1$
```

Comment:
Looks like it will not, but I think it will not be supported by SWT. Anyway this is interesting question - what will be used as SWT internal browser for Windows 10. My guess is IE 11, but need to check.